modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -589,29 +594,69 @@ export default class Tileset3D {
     this.stats.get(POINTS_COUNT).count = pointsRenderable;
   }
 
-  _initializeTileSet(tilesetJson) {
+  async _initializeTileSet(tilesetJson) {
```

Comment:
Add types?

Reply:
There should be `I3STilesetHeader` or `Tile3DTilesetHeader` then. Is it correct if we add `@loaders.gl/i3s` and `@loaders.gl/3d-tiles` as dependencies in this module? The existing architecture is that "Loader" comes within this `tilesetJson` in constructor so we don't have that dependencies on `@loaders.gl/i3s` and `@loaders.gl/3d-tiles`